pkg/cmd/operator/operator.go

```diff
@@ -108,6 +125,7 @@ func NewOperatorCmd(streams genericclioptions.IOStreams) *cobra.Command {
 
 	cmd.Flags().IntVarP(&o.ConcurrentSyncs, "concurrent-syncs", "", o.ConcurrentSyncs, "The number of ScyllaCluster objects that are allowed to sync concurrently.")
 	cmd.Flags().StringVarP(&o.OperatorImage, "image", "", o.OperatorImage, "Image of the operator used.")
```

Comment:
It should stay. It's a way to disable the auto detection or run this in development with different image, especially if you run the operator locally (not in a Pod) where it can't detect the status.